src/server/session.ts

```diff
@@ -1641,7 +1641,7 @@ namespace ts.server {
         private getDocCommentTemplate(args: protocol.FileLocationRequestArgs) {
             const { file, languageService } = this.getFileAndLanguageServiceForSyntacticOperation(args);
             const position = this.getPositionInFile(args, file);
-            return languageService.getDocCommentTemplateAtPosition(file, position);
+            return languageService.getDocCommentTemplateAtPosition(file, position, { generateReturn: this.getPreferences(file).generateReturnInDocTemplate });

```

Comment:
Prefer the name in LS to be same so we can just pass `this.getPreferences()` instead of having to create separate object

Reply:
@sheetalkamat Thanks for the feedback. Renamed `generateReturn` to `generateReturnInDocTemplate`.